sprints/dashboard/libs/jira.py

```diff
@@ -314,8 +314,11 @@ def poker_session_vote_values(self, board_id: int) -> list[float]:
         :return: A list of possible vote values.
         """
         response = self._get_json(f'board/{board_id}/settings', base=self.AGILE_POKER_URL)
-        print(response)
-        return [float(vote['value']) for vote in response.get('voteValues', [])]
+        return [
+            float(vote['value'])
+            for vote in response.get('voteValues', [])
+            if vote['value'].replace('.', '', 1).isdigit()
```

Comment:
This feel like a hacky heuristic. It will fail in cases like `1..1`.  Perhaps something like this will be more reliable but still allow a nice list comprehension:

```
def is_number(n: str) -> bool:
  try:
    float(n)
  except:
    return False
  return True

... [float(n) for n in values if is_number(n)]
```

Reply:
Oh wait, my bad, missed that this was replacing it only once.  :grimacing: 

Still, this kind of indicates that it's not clear exactly what it's doing and how.